packages/server/package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "@botpress/messaging-base": "0.0.1",
     "@botpress/messaging-engine": "0.0.1",
+    "@botpress/messaging-channels": "0.0.1",
```

Comment:
We should consider using '*' as the version since it references another package and we want to use it whatever version it is at